### PR TITLE
Add default branding for new app

### LINKED ISSUE #446: Support injecting default email sender and SMS sender at app creation
In https://github.com/authgear/authgear-server/pull/387 we added support for provide SMTP credentials. But we still need to provide a default email sender and SMS sender for the whole thing to work.

### PR DESCRIPTION
**Pull request by louischan-oursky**
fix #446

### CODE REVIEW

**Review by kiootic — APPROVED** [review]
Do we need to customize `app.app-name` as well?

**Comment by louischan-oursky** [comment]
Good idea. Let me add it too.

**Comment by louischan-oursky** [comment]
Updated.

File: .env.example [hunk 1]
```diff
@@ -10,3 +10,5 @@ APP_HOST_TEMPLATE={{ .AppID }}.localhost:3000
 APP_SECRET_DATABASE_URL=postgres://postgres:postgres@localhost:5432/postgres?sslmode=disable
 APP_SECRET_DATABASE_SCHEMA=public
 APP_SECRET_REDIS_URL=redis://localhost
+APP_BRANDING_EMAIL_DEFAULT_SENDER=user@example.com
```
Comment (kiootic) [inline .env.example:13]: Set default app name here?

**Review by kiootic — COMMENTED** [review]
@louischan-oursky conflicted

**Comment by louischan-oursky** [comment]
rebased and add the default value to `.env.example`.

**Review by kiootic — APPROVED** [review]

**Pull request merged**